src/test/javascript/spec/interceptor/errorhandler.interceptor.spec.ts

```diff
@@ -0,0 +1,60 @@
+import { HttpErrorResponse } from '@angular/common/http';
+import { throwError } from 'rxjs';
+import { ErrorHandlerInterceptor } from 'app/core/interceptor/errorhandler.interceptor';
+import { EventManager } from 'app/core/util/event-manager.service';
+
+describe(`ErrorHandlerInterceptor`, () => {
+    let errorHandlerInterceptor: ErrorHandlerInterceptor;
+
+    let eventManagerMock: EventManager;
+
+    beforeEach(() => {
+        eventManagerMock = {
+            broadcast: jest.fn(),
+        } as any as EventManager;
+
+        errorHandlerInterceptor = new ErrorHandlerInterceptor(eventManagerMock);
+    });
+
+    afterEach(() => {
+        jest.restoreAllMocks();
+    });
+
+    it('should broadcast http errors', () => {
+        const error = new HttpErrorResponse({ status: 400 });
+        const mockHandler = {
+            handle: () => throwError(() => error),
+        };
+
+        errorHandlerInterceptor.intercept(null as any, mockHandler).subscribe();
+
+        expect(eventManagerMock.broadcast).toHaveBeenCalledWith({
+            name: 'artemisApp.httpError',
+            content: error,
+        });
+    });
+
+    it('should not broadcast an http error if status is 401 and message is empty', () => {
```

Comment:
I think it makes sense to add a positive 401 check as well. Right now these tests would also succeed if a 401 never gets broadcasted

Reply:
I'll let the first test check a few status codes